### PR TITLE
USHIFT-4127: Refactor bootc RPM repository configuration

### PR DESCRIPTION
**Pull request by ggiguash**
This is a first step for implementing various upgrade scenarios. 
The new technique should make it more straightforward to set up repositories when building container images.

### CODE REVIEW

File: test/image-blueprints/layer5-bootc/group1/cos9-bootc-source.containerfile [hunk 1]
```diff
@@ -1,26 +1,23 @@
...
 # Copy the MicroShift repository contents
 COPY ./rpm-repos/$USHIFT_RPM_REPO_NAME $USHIFT_RPM_REPO_PATH
 
-# Add the following repositories and print their contents:
-# - MicroShift local RPM repository
-# - OpenShift Mirror Beta previous minor version repository for MicroShift dependencies
-# Install MicroShift and cleanup
-RUN ${REPO_CONFIG_SCRIPT} -create ${USHIFT_RPM_REPO_PATH} && \
+# Copy repository configuration
+COPY ./bootc-images/microshift-local.repo ./bootc-images/centos9-nfv.repo ./bootc-images/rhocp-y.repo \
```
Comment (pmtk) [inline test/image-blueprints/layer5-bootc/group1/cos9-bootc-source.containerfile:11]: Would this work?
```suggestion
COPY ./bootc-images/$USHIFT_RPM_REPO_NAME.repo ./bootc-images/centos9-nfv.repo ./bootc-images/rhocp-y.repo \
```

File: test/image-blueprints/layer5-bootc/group1/cos9-bootc-source.containerfile [hunk 2]
```diff
@@ -1,26 +1,23 @@
...
+
+# Print repository configuration contents.
+# Install MicroShift and cleanup.
+RUN dnf repoinfo --enabled && \
     dnf install -y firewalld "microshift-{{ .Env.SOURCE_VERSION }}" "microshift-release-info-{{ .Env.SOURCE_VERSION }}" && \
     systemctl enable microshift && \
-    ${REPO_CONFIG_SCRIPT} -delete && \
-    rm -f ${REPO_CONFIG_SCRIPT} && \
-    rm -rf $USHIFT_RPM_REPO_PATH && \
+    rm -vf /etc/yum.repos.d/microshift-local.repo /etc/yum.repos.d/centos9-nfv.repo /etc/yum.repos.d/rhocp-y.repo && \
```
Comment (pmtk) [inline test/image-blueprints/layer5-bootc/group1/cos9-bootc-source.containerfile:19]: Maybe we could use some prefix for our repositories to remove them with glob? Something like `rm -vf /etc/yum.repos.d/test*.repo`

Comment (ggiguash) [reply test/image-blueprints/layer5-bootc/group1/cos9-bootc-source.containerfile:19]: Let me try the `microshift-` prefix.

**Pull request merged**